docs/settings/alert-action-settings.asciidoc

```diff
@@ -207,3 +207,16 @@ Specifies the behavior when a new or changed rule has a schedule interval less t
 
 `xpack.alerting.rules.execution.actions.max`::
 Specifies the maximum number of actions that a rule can trigger each time detection checks run.
+
+`xpack.alerting.rules.execution.actions.connectorTypeOverrides`::
+Overrides the configs under `xpack.alerting.rules.execution.actions` for the connector type with the given ID. The value is array of objects consists of the key(s) that are wanted to be overridden under actions object and the id of the specific connector type that is wanted to be modified.
```

Comment:
Per https://github.com/elastic/kibana/pull/128294/files#r833808929 and https://github.com/elastic/kibana/pull/128294/files#r833810123, IMO this can be simplified:
```suggestion
Overrides `xpack.alerting.rules.actions.*` settings for a specific type of connector. List the connector type identifier and its settings in array of objects.
```

Reply:
done